### PR TITLE
Fix API change

### PR DESCRIPTION
**Pull request by codinghipster**
Fix for Issue #5

### CODE REVIEW

**Comment by tomaszmrugalski** [comment]
Thanks for submitting this PR. I confirm it solves the ImportError exception. However, it doesn't work for the orbit I'm working with. This used to work with poliastro 0.13.1 and tle-tools 0.2.1.

This is a rather boring, 850km, almost circular polar orbit. Nothing crazy here.

Here's the minimal code to reproduce the problem:
```
from tletools import TLE

tle_txt = """
NOAA 18
1 28654U 05018A   20098.54037539  .00000075  00000-0  65128-4 0  9992
2 28654  99.0522 154.2797 0015184  73.2195 287.0641 14.12501077766909
"""

tle_lines = tle_txt.strip().splitlines()

# This seems to work
tle = TLE.from_lines(*tle_lines)

# This crashes with AssertionError about M not being within -pi .. pi range
orb = tle.to_orbit()
```

I'll try to debug the problem.

File: tletools/tle.py [hunk 1]
```diff
@@ -162,7 +161,8 @@ def a(self):
     def nu(self):
         """True anomaly."""
         if self._nu is None:
-            self._nu = _M_to_nu(self.M * DEG2RAD, self.ecc) * RAD2DEG
+            M = self.M * DEG2RAD
```
Comment (astrojuanlu) [inline tletools/tle.py:164]:
```suggestion
            M = ((self.M + 180) % 360 - 180) * DEG2RAD
```

> Fields 2.3, 2.4, 2.6, and 2.7 all have units of degrees and can range from 0 up to 360 degrees

https://celestrak.com/columns/v04n03/#FAQ01

see https://github.com/FedericoStra/tletools/pull/6#issuecomment-628876160 by @tomaszmrugalski

Comment (tomaszmrugalski) [reply tletools/tle.py:164]: The fix @astrojuanlu proposed works. I figured it out on my own, but Juan was faster :) I've created #7 (which has cherry-picked this change + Juan's fix + extra unit-tests)

**Comment by FedericoStra** [comment]
Ok, so in the end I decided for the other PR, mainly because it had also some tests with it. I also added the same computation to the class `TLEu`, which was missing.

**Pull request closed without merging**